grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -379,7 +380,7 @@ class ReportService implements ApplicationContextAware {
             SELECT 0, transaction_type.transaction_code, substring_index(transaction_type.name, '|', 1), transaction_type.id
             FROM transaction_type
         """
-        executeStatements([insertStatement])
+        dataService.executeStatements([insertStatement])
```

Comment:
What the ...?

Reply:
How did this happen? Maybe a bad rebase?

Reply:
I am not sure if it's rebase fault, or what, since it's here from the first migration commit.